docs/codebase-overview.md

```diff
@@ -7,12 +7,12 @@ If you want to [contribute](CONTRIBUTING.md), we hope that this overview will he
 ## Top-Level folders
 
 * `components` - React components
-    * `[COMPONENT]` - components that are part of the public API
-        * `__tests__` - Mocha and Jest tests
-        * `__examples__` Examples used in storybook and documentation website
-        * `__docs__` - storybook and documentation website imports of examples
-        * `private` - components that are not a part of the public API
-    * `utilities` - components that are shared between other components (not public API)
+  * `[COMPONENT]` - components that are part of the public API
+    * `__tests__` - Mocha and Jest tests
+    * `__examples__` Examples used in storybook and documentation website
+    * `__docs__` - storybook and documentation website imports of examples
+    * `private` - components that are not a part of the public API
+  * `utilities` - components that are shared between other components (not public API)
```

Comment:
Shouldn't `presets` be added to this list?

Reply:
Yep. Added.